src/language-js/traverse/visitor-keys.evaluate.js

```diff
@@ -29,6 +29,8 @@ const additionalVisitorKeys = {
   BigIntTypeAnnotation: [],
   // This may invalid, need investigate
   TSAbstractMethodDefinition: ["decorators"],
+  // Babel missing this
+  Program: ["interpreter"],
```

Comment:
What is this property supposed to contain? A node?

Reply:
Yes, a node called `InterpreterDirective`. https://github.com/fisker/prettier/blob/bf3ed96a84017fade16bc5e1e82bfe55a567b870/src/language-js/printer-estree.js#L207

https://github.com/fisker/prettier/blob/bf3ed96a84017fade16bc5e1e82bfe55a567b870/src/language-js/printer-estree.js#L785